src/main/kotlin/interfaces/BlackJackController.kt

```diff
@@ -0,0 +1,73 @@
+package interfaces
+
+import application.BlackJackGame
+import common.Executable
+import domain.Participants
+import domain.Player
+import interfaces.ui.CardInfo
+import interfaces.ui.InputConsole
+import interfaces.ui.OutputConsole
+
+class BlackJackController : Executable {
+
+    override fun execute() {
+
+        val participants = queryParticipants()
+        val blackJackGame = BlackJackGame(players = participants)
+
+        initGame(blackJackGame)
+        showAllParticipantsHands(blackJackGame)
+
+        queryReceiveCard(blackJackGame)
+
+        showFinalResult(blackJackGame)
+    }
+
+    private fun initGame(blackJackGame: BlackJackGame) {
+        blackJackGame.init()
+        val playerNames = blackJackGame.allPlayers().map { it.name }
+        OutputConsole.printInit(playerNames = playerNames)
```

Comment:
```suggestion
        OutputConsole.printInit(players =  blackJackGame.allPlayers())
```
BlackJackController가 조금 커진거 같아요! 
뷰에 대한 코드들이 조금있는거 같은데 더분리하면 좋을거같아요! 
이런 부분은 OutView에 있어 될거같아요!

Reply:
해당 코드는 view가 player에 대한 의존성이 생기는데요...!!